ecs-cli/modules/commands/flags/flags.go

```diff
@@ -57,6 +57,10 @@ const (
 	TimeStampsFlag     = "timestamps"
 	CreateLogsFlag     = "create-log-groups"
 
+	// Service Discovery
```

Comment:
is there a reason the `vpc-id` flag isn't here?

Reply:
There's an existing one for cluster up which I just re-used.